### ISSUE TITLE
Badging and colour

### ISSUE THREAD

**Issue by ajburls** [dwyl/best-evidence#36]
As the product owner
I would like the application to follow City University branding guidelines (the minimum requirement is to use City colours and the logo)
In order for it to appeal to my stakeholders

[written by Mark to replace lack of description - the above is a summary of what Amanda said during sprint planning]

**Comment by ajburls** [comment]
@SimonLab Haven't heard back from City University about the designer they offered but have got pdf of their branding guidance and colours.
[City UoL brand guidelines September 2016.pdf](https://github.com/dwyl/best-evidence/files/1162861/City.UoL.brand.guidelines.September.2016.pdf)

**Comment by markwilliamfirth** [comment]
City red hex: #C00F22

**Comment by SimonLab** [comment]
Thanks for the pdf @ajburls .
Would you like us to discuss a future design with @harrygfox soon? Are you happy for us to discuss this for sprint 2?

**Comment by ajburls** [comment]
@SimonLab Yes, please. Not sure when City designer will contact me. I will chase if I haven't heard by end of Monday. But it may not be helpful anyway. If something comes of it, results can feed into Sprint 2 and discussions with @harrygfox

**Comment by markwilliamfirth** [comment]
From designer:

Hi Amanda,

I've now had a play and a look at the wire frames. It's all looking good and as it's all so clearly laid out I think the only thing I would suggest is the use of our City colours. I have attached our brand guidelines where hex references can be found for the colours on page 18, though advise speaking to Cara Perry (digital designer) or Dimitra Bazani (User Experience Architect) as I believe they may have slightly different values based on usability for the website.

To ensure it's 'City' I would use the City red as the base colour, as per example below, though if it is green for a specific reason currently then red isn't essential.
I've also attached a screenshot of the responsive version of the website which uses our responsive logo in the header. If you do not wish to use the logo so prominently I suggest placing it in a footer in the responsive form (if possible) and in full on the 'about us' page. I've attached both versions. The full version of the logo has white space built in as per our guidelines.

I spotted our old name is used on the about us page currently, which will need updating to City, University of London.

![unnamed](https://user-images.githubusercontent.com/5732209/28977088-88a48dd2-7938-11e7-9bde-0bbb85abe63b.png)


 (this is very rough and colours are inaccurate. Would be the Pantone 1797 and 5115).

![unnamed 1](https://user-images.githubusercontent.com/5732209/28977078-86263650-7938-11e7-9584-dc247006c6ae.jpg)



City's font is Meta OT. If you wish to use it you will have to source licensing for it as the only license we have is for print. It is used on our website under a different license which you may be able to utilise. For that I suggest speaking to the web development lead, Tom Waddington.

I hope that's all of some help and good luck with the final sprint!

Thanks,

Jon



![city uol logo rgb dk1a](https://user-images.githubusercontent.com/5732209/28977113-a77828d6-7938-11e7-9785-6b981c6ee259.jpg)
[City UoL brand guidelines v1.3.pdf](https://github.com/dwyl/best-evidence/files/1200976/City.UoL.brand.guidelines.v1.3.pdf)
![city uol logo_responsive rgb dk1a](https://user-images.githubusercontent.com/5732209/28977115-a78492c4-7938-11e7-8afa-fc7682f2096f.jpg)

**Comment by markwilliamfirth** [comment]
@Cleop please can you provide a time estimate to change the hex codes?

**Comment by markwilliamfirth** [comment]
@waddie is there a way we can use City's font (Meta OT) for this app?

**Comment by waddie** [comment]
@markwilliamfirth Licensing for FF Meta is a one-time purchase per domain based on page views per month. We have a licence for the complete FF Meta collection on city.ac.uk

If this will be hosted on a *.city.ac.uk and likely to be serving <10k page views per month, you'll be able to use our existing licence, otherwise you'll need a new licence.

**Comment by markwilliamfirth** [comment]
Thanks @waddie 

Linked to #30 

@ajburls please let us know how you'd like us to proceed with the font / domain

**Comment by Cleop** [comment]
Color changes can be made with this information.

As well as font/typeface info, we also need more information on how/where the logo should be used from @ajburls before I can implement that.

**Comment by ajburls** [comment]
Logo should just be on the about us page not on main page.

**Comment by markwilliamfirth** [comment]
Thanks @ajburls 

To confirm the following needs to be done:

- [ ] Hex codes / colours need to be changed to city colours
- [ ] City's responsive logo needs to be added to the about us page (waiting on #23 )

@ajburls please could you assign a priority label to this issue? Due to the fact that you are opting for a different domain #30 to a city domain, I assume you wish the font to stay as it is?

**Comment by ajburls** [comment]
@Cleop City powerpoint 
[City.pptx](https://github.com/dwyl/best-evidence/files/1218520/City.pptx)

**Comment by markwilliamfirth** [comment]
@ajburls please can you test and close this issue if you are happy?

**Comment by ajburls** [comment]
@cleop The errors on this page now corrected and the links in the right place. So this page can be left as it is.

However, I think the text should be wider so that it fits on page without need to scroll and it would be better for City logo to come after text on City. I have changed priority as this is a lower priority.

**Comment by markwilliamfirth** [comment]
@ajburls I have recorded this new requirement here:
https://github.com/dwyl/best-evidence/issues/172

So this issue can now be closed if you are happy what was implemented
